File: Java/프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        String answer = "";
        
        for(int i=0; i<my_string.length(); i++) {
            char a = my_string.charAt(i);
            
            if(Character.isLowerCase(a)) {
               a = Character.toUpperCase(a);
                
            }
            else
               a = Character.toLowerCase(a);  
              answer += a;
        } 
      
       
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.js
function solution(my_string) {
    var answer = '';
    
    for(let i=0; i<my_string.length; i++){
        if(my_string.charAt(i) === my_string.charAt(i).toLowerCase())
            answer += my_string.charAt(i).toUpperCase();
        else
            answer+= my_string.charAt(i).toLowerCase();
    }
    return answer;
}